tensorflow_addons/image/filters.py

```diff
@@ -112,6 +109,66 @@ def mean_filter2d(image, filter_shape=(3, 3), name=None):
         return y
 
 
+@tf.function
+def median_filter1d(image,filter_shape=3, name=None):
```

Comment:
I think `median_filter1d(image, filter_shape=3)` should have the same functionality with `median_filter(image, filter_shape=(3, 1))`? If so, can we just call `median_filter2d` in `median_filter1d`?

Reply:
1d here just mean we are using a 1d filters. Hence the filter-shape is a single odd integer.  Not a tuple.  1d filter will improve time complexity im case of batch processing.

Reply:
Okay seems the functionality is quite different with the one in my understanding.